pulp_rpm/app/viewsets.py

```diff
@@ -224,6 +229,12 @@ class RpmRepositoryViewSet(RepositoryViewSet, ModifyRepositoryActionMixin, Roles
             "rpm.sync_rpmrepository",
             "rpm.view_rpmrepository",
         ],
+        "rpm.rpmrepository_content_manager": [
+            "rpm.delete_rpmrepository_version",
```

Comment:
I might have confused you. I meant to see the delete_repo_verison in the owner role.
If you meant to add a separate role for content_manager  then you also miss repair permission in that list.